commons/data_access_layer/cosmos_db.py

```diff
@@ -225,6 +232,7 @@ def find(
     def find_all(
         self,
         event_context: EventContext,
+        only_active = True,
```

Comment:
Comparing this with `visible_only` parameter. It would be good to rename this var to `active_only`.

Reply:
Ok, it has been adjusted, with a new approach to avoid the only_active argument.